lib/services/auth.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:brew_crew/models/user.dart';
+import 'package:brew_crew/services/database.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class AuthService {
+  final FirebaseAuth _auth = FirebaseAuth.instance;
+
+  // create user obj based on firebase User
+
+  TheUser _userFromFirebaseUser(User user) {
+    return user != null ? TheUser(uid: user.uid) : null;
+  }
+
+  //auth change user stream
+
+  Stream<TheUser> get user {
+    return _auth
+        .authStateChanges()
+        //.map((User user) => _userFromFirebaseUser(user));
+        .map(_userFromFirebaseUser);
+  }
+
+  // sign in anon
```

Comment:
po co te komentarze? xD
czy coś w działaniu tej metody jest niejasnego/niezwykłego? jeśli nie to najlepszym komentarzem jest sam kod

Reply:
już usunięte szefuncio, było to po to bym pamiętał jakie operacje mam zrobić tuż po utworzeniu pliku auth.dart